synnax/pkg/api/channel_test.go

```diff
@@ -1,242 +0,0 @@
-// Copyright 2023 Synnax Labs, Inc.
```

Comment:
We decided that we're not going to test at the API level since it's hard to setup and most of the endpoints are hit by client tests anyways. On a second thought I think putting this back in the codebase can't hurt so I'm going to reinstate these test cases.

Reply:
OK I remember why we took it out now – it was because it's hard to incorporate token & user info into a request made from the unit test so the tests were broken after auth was added. Going to remove these tests.